schema.sql

```diff
@@ -0,0 +1,140 @@
+CREATE DATABASE IF NOT EXISTS `codestroke`;
+USE `codestroke`;
+
+CREATE TABLE IF NOT EXISTS `cases` (
+  `case_id` int NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `first_name` varchar(30) NOT NULL,
+  `last_name` varchar(30) NOT NULL,
+  `dob` date NOT NULL,
+  `address` text NOT NULL,
+  `gender` varchar(1) NOT NULL,
+  `last_well` datetime NOT NULL,
+  `nok` varchar(40) NOT NULL,
+  `nok_phone` varchar(16) NOT NULL,
+  `medicare_no` varchar(12) DEFAULT NULL,
+  `status` tinyint DEFAULT 0
+);
+
+CREATE TABLE IF NOT EXISTS `case_hospitals` (
+  `id` int NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `case_id` int NOT NULL,
+  `hospital_id` int NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS `case_histories` (
+  `case_id` int NOT NULL PRIMARY KEY,
+  `pmhx` text NOT NULL,
+  `meds` text NOT NULL,
+  `anticoags` bool NOT NULL,
+  `anticoags_last_dose` datetime DEFAULT NULL,
+  `hopc` text NOT NULL,
+  `weight` float DEFAULT NULL,
+  `last_meal` datetime DEFAULT NULL
+);
+
+CREATE TABLE IF NOT EXISTS `case_assessments` (
+  `case_id` int NOT NULL PRIMARY KEY,
+  `facial_droop` bool DEFAULT NULL,
+  `arm_drift` bool DEFAULT NULL,
+  `weak_grip` bool DEFAULT NULL,
+  `speech_difficulty` bool DEFAULT NULL,
+  `bp_systolic` int DEFAULT NULL,
+  `bp_diastolic` int DEFAULT NULL,
```

Comment:
I saw this, but I thought it should be possible to split at the front end level since it seems to make more sense to me have numerical data stored as a numeric type (e.g. if you're going to compare blood pressures/plot vitals etc.). If not though, I can make it a string for now if that's easier?